### ISSUE TITLE
Failed to import - no ship found "Viper_MkIV"

### ISSUE THREAD

**Issue by Fireforge** [EDCD/coriolis#42]
I got an `"No such ship found: \"Viper_MkIV\""` error when trying to import my Viper Mk IV straight from the E:D Market Connector. It seems the names given by that app don't follow the conventions of your database. You could close this and say it's their issue, not yours.

That said, you might consider stripping out any and all underscores in the ship names before checking for equality here: https://github.com/cmmcleod/coriolis/blob/master/src/app/components/ModalImport.jsx#L197
I don't think that operation would ever match it to the wrong ship, but maybe I'm wrong.

Details below:
```
Browser: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.99 Safari/537.36
Path:
Error: Unknown
Details:
scriptUrl: /import?data=H4sIAE7iTFgC_82W32_bIBDH_xeeowkw_pW3Nl1_TI1UzVX3ME0WiUmLSsDCuFNW9X8f2I1nmrQjah-aJ47A5XPfOy73CJZU3yowfbSLmi652YApfJqAlWYMTFdUNMwaLRPuyJpy6R9FiT2rWcP0A3vh5AtJnuyXvAJTPAFrVbWCNe7MkV6rVnf-uk238n7OXUE4S1KM43QCJF3b78ANr5ku5_cXN2XvoTzTtGIITICyVEa39mqtudI92QS0UigqmfUGJ-CBipa52CzTsVosBIMITH_-GizsWZFnEc-KPSvxrNSzMs_KxxaCvXXCllSg0RqP1s8UX-Utl2ymRKeb2znVVpTijq_Miea99G-LCROCUDKIeSFNeb6xglbuelnwP4yUM0GbhryiJ9yvZxwlSR45UU9tkVxTeR_EEpHcY9leHpNEIZnFJI3IQNNZDuaSr1jR1rXSJkwbEnk8o_sdEu6RgooNEZINRM5wQHP7dPo0NkFAMEMeUH_3QxI1ZxVv1-dUV7Xi0qAAHpITSAae89qUV609cEntwz9udWPKM75eUFEWayrEYWBpSuAeLPxJsK5s4sw3tejf3ZWw5wzVm6O61oou74qWGxbUyyKUehntXfl-AqorhqOcwh5R_Wb6hDdG80VrlA6r-CzzcV446UotOqDsCYyjgcwZA5qLNOwZQpLsQtXuekjlo_9V_nda0TB1MPbVKZhslG4-RpRCKANRH1EYTerLYhs_E9UZk0zTbare2xU6KLyFcrXe7UR9yKMd0jfE0U683Qlp_DDdE8qMCXG8bf_PvTY-MJIkJukQSNJ5Cmhtif0QsvNfVCyVqnsfb2f7FZoI5h2L6zuHdto491ravBWGz6iUSr6jnVnfON5Fwp8B6ZrLzSEiJQmOI-wRzTS389DSIrH-OcC3Z4hXkVCW5ztMz8X-g9FayX9j2NOe2RQMvh7BXWslcaWQovEEjAm0U0E8JohImtkx4S_JPTgljgsAAA%3D%3D
line: 0
col: 0
error: "No such ship found: \"Viper_MkIV\""
```

**Comment by cmdrmcdonald** [comment]
Thank you for reporting this.  It has been fixed, and will turn up in the next release of Coriolis.  If you want to check it you can re-run your import from EDMC and then change the URL from https://coriolis.edcd.io/ to http://beta.coriolis.edcd.io/

**Comment by Fireforge** [comment]
I just tried it again, and the import failed, however the error message is different this time: `line:0 col:0 error: {}`.

Details from the beta page:
```
Browser: Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.99 Safari/537.36
Path:
Error: Unknown
Details:
scriptUrl: /import?data=H4sIALPKUFgC_82X30_bMBDH_5c8V5NjO07SNyjjx0QlRNH2ME2R27hg4dqR4zAxxP8-O6ZZvRbmDiR4ai_xnT_5-u5yeUgWVF-rZPxg_zR0wc19Mk7J4yhZasaS8ZKKllmjY8KtWVEud6zVrGX6joV3wCdMHu1NXidjOEpWqu4Ea92aA71Sne7j9Rfdv2A755LCguQQZvkokXRl7yVfecN0Nb09-1r5CNWJpjVLk1GiLJXRnXVtNFfak42STgpFJbPRwCi5o6KzYYBjOlTzuWAgTcbffwwWDCwUWDiwssAigZUHVhFY5aaVAm8dsQUVaYQaJM9RMajRu1UTtZpT434aZpg0_6FFHwi-8_7o_fb_LK-5ZBMl-px0J3Ks7RazG740R5r7tH4ZDRCcpmRAO5OmOr23yVo792rGfzFcTQRtW_wMH9jNlyFCSuQgj20BXlF5G8WCcBmwrJ03SVCMUhDnCA80veVgzvmSzbqmUdrEaYNRwLPh3yNBjxRVyCnGxUDkDAc0tW3JH2MbBQSKNADyvm9yUFNW8251SnXdKC5NRF0DXGKAB57TxlQXnV1wTm1TPex0a6oTbrNcVLMVFWI_sDzHYAcW_CBYF_bgzBc193V3Iew6Q_X9QdNoRRc3s44bFvWeQGkenKgPFcaJyK4MbJwp8IjqJ9NHvDWazzujdFzGF0WI81eQPtXQHmmPQYYGMmcMaO5J48oQYLIN1Tj3mMxP_5X5l7SmcepAGKozY7JVun0bUWZCGZD6J4qjyUNZbONnoj5hkmm6PqrXdoUeCu4BhXDYoSZuSruki6CH7zn5IIxgMdAgL7evvP4K9s04jg_kO0SbMCEO1y-ap66e7akZyXA-QGZrpAGS9FeixiVCMN56D84WSjU-xssiPsOHQNnTuZ63b5fPyqCdTjth-IRKqeQrWqmNDbNtJPgRkK64vN9HJEJghmBANNHczmILi8R8KYKX55dnkdKiLLeYnjLrG6ONkn9GwMcd3xzJEOshuemsJC4V8nTzywZilNkZdZMA4bywNfcbcMHMeWcNAAA%3D
line: 0
col: 0
error: {}
```
URL I used: `http://beta.coriolis.edcd.io`
Version on Beta page: `2.2.6b - 2016-12-13`

**Comment by cmdrmcdonald** [comment]
Apologies, this was an unrelated issue due to beta testing.

Please try it again now.  Note that you might need to bring up the site first, then hit the 'refresh to update' banner at the top, then retry the import.

**Comment by cmdrmcdonald** [comment]
Fixed in 2.2.6
